src/status_im/ui/screens/offline_messaging_settings/views.cljs

```diff
@@ -3,6 +3,7 @@
             [status-im.i18n :as i18n]
             [status-im.ui.components.common.common :as common]
             [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [status-im.ui.components.list.views :as list]
             [status-im.ui.components.react :as react]
             [status-im.ui.components.renderers.renderers :as renderers]
```

Comment:
Can be removed?

Reply:
right, i'm removing unnecessary requires..